src/screens/Pagination.js

```diff
@@ -139,6 +139,33 @@ const PaginationPage = () => (
             <Example defaultValue> 10 </Example>
           </PropertyValue>
         </Property>
+
+        <Property name="stepOptions">
+          <Description>
+            The option for a user to select the number of items per page that
+            they would like to view.
```

Comment:
Just wanted to give a little more description to devs for what happens when this prop is `true`
```suggestion
            Display a Select component where the user can choose the number 
            of items per page that they would like to view.
```